lib/ansible/modules/monitoring/monit.py

```diff
@@ -23,26 +23,31 @@
                     'status': ['preview'],
                     'supported_by': 'community'}
 
-
 DOCUMENTATION = '''
 ---
 module: monit
 short_description: Manage the state of a program monitored via Monit
 description:
      - Manage the state of a program monitored via I(Monit)
-version_added: "1.2"
+version_added: "1.3"
 options:
   name:
     description:
       - The name of the I(monit) program/process to manage
-    required: true
+    required: false
+    default: null
+  service:
+    description:
+      - The name of the I(monit) program/process to manage. This is an alias of B(name).
+    required: false
     default: null
+    version_added: "2.4"
   state:
     description:
       - The state of service
     required: true
     default: null
-    choices: [ "present", "started", "stopped", "restarted", "monitored", "unmonitored", "reloaded" ]
+    choices: [ "present", "started", "stopped", "restarted", "monitored", "unmonitored", "reloaded", "deployed" ]
```

Comment:
I don't think installation should be covered by this modules, there are plenty of package management modules that are better suited for this, at worst a role that downloads and installs it